majifix311/src/main/java/com/example/majifix311/EventHandler.java

```diff
@@ -46,4 +50,24 @@ public static void sendMyReportedProblemsList(Context context, ArrayList<Problem
         LocalBroadcastManager.getInstance(context).sendBroadcast(resultIntent);
     }
 
+    public static void retrievedMyRequests(Context context, ArrayList<Problem> problems){
+        System.out.println("Received requests from server");
+
+        Intent resultIntent = new Intent();
+        resultIntent.setAction(BROADCAST_MY_PROBLEMS_FETCHED);
+        resultIntent.putExtra(IS_SUCCESS, true);
+        resultIntent.putExtra(REQUEST_LIST, problems);
+        LocalBroadcastManager.getInstance(context).sendBroadcast(resultIntent);
+    }
+
+    public static void retrievedMyRequests(Context context, Throwable error){
+        System.out.println("Erred on request from server");
+
+        Intent resultIntent = new Intent();
+        resultIntent.setAction(BROADCAST_MY_PROBLEMS_FETCHED);
+        resultIntent.putExtra(IS_SUCCESS, false);
+        resultIntent.putExtra(REQUEST_LIST, error);
```

Comment:
In the future, let's architect so that errors have a regular intent, so it can be used more generically if required. For now, as this will be refactored anyway in the next task, no biggie.